admin/commands/storage/backfill_tx_error_messages.go

```diff
@@ -59,7 +59,7 @@ func (b *BackfillTxErrorMessagesCommand) Validator(request *admin.CommandRequest
 
 	sealed, err := b.state.Sealed().Head()
 	if err != nil {
-		return fmt.Errorf("failed to lookup sealed header: %w", err)
```

Comment:
the original error message was clearer. let's keep that here and in other cases.